Formula/topshell.rb

```diff
@@ -0,0 +1,33 @@
+require "language/node"
+
+class Topshell < Formula
+  desc "Purely functional, reactive scripting language"
+  homepage "http://show.ahnfelt.net/topshell/"
+  url "https://github.com/topshell-language/topshell/archive/v0.7.7.tar.gz"
+  sha256 "a24c759b231fbd5aaed3409ed5d12e98ae2987cc27672fef27964a21017dccd6"
+
+  depends_on "node@8" => :build
```

Comment:
We prefer not to accept new formulas that do not build with the latest version of compilers and tools, as it typically indicates lack of long-term support. In this case, `node@8` depends on Python 2, which will become unsupported on 2019-12-31, and will be removed from Homebrew. Can you migrate to latest `node`?